### ISSUE TITLE
Pre-training Loss curves of VJEPA2

### ISSUE THREAD

**Issue by ajamal-isrg** [facebookresearch/vjepa2#44]
Thank you for open-sourcing VJEPA2 — it’s an incredible resource. I’m currently exploring the model and was wondering if you could share the pre-training loss curves. Were there any notable trends or behaviors you observed during pre-training that might guide further experimentation?

Thanks in advance for your help!

**Comment by luowyang** [comment]
See also #56, which shows some typical loss curves we have encountered during pretraining.

**Comment by ajamal-isrg** [comment]
Interesting, I think I have also seen the similar trend when training VJEPA. The loss increases after it has decreased for a fewer number of epochs. 
See https://github.com/facebookresearch/jepa/issues/68 and https://github.com/facebookresearch/jepa/issues/85

**Comment by ballasnicolas** [comment]
yes, that is correct. Note that we only have access to a 'proxy' loss function due to the use of cut-gradient and EMA in our approach. Potentially, one could train similar model with VICreg type of loss instead.
